tests/basic/pure_test.go

```diff
@@ -339,3 +339,88 @@ var _ = Describe("{BringUpLargePodsVerifyNoPanic}", func() {
 		AfterEachTest(contexts, testrailID, runID)
 	})
 })
+
+// This test validates volume token timeout for FADA-volumes
+var _ = Describe("{FADAVolTokenTimout}", func() {
+	/*
+					https://portworx.atlassian.net/browse/PTX-18941
+
+					PWX :
+					https://portworx.atlassian.net/browse/PWX-33632
+
+					Bug Description :
+						The current token manager timeout is 3 minutes. This is not sufficient for FADA volumes since as part of FADA operations a REST call is made to FA.
+		                We have seen slowness in these APIs taking upto 15s to complete. This causes the token timeout to hit and PX to panic.
+
+			1. Deploying nginx pods using two FADA volumes with volume placement strategy, creating volumes on  node-1
+		    2. Deploy nginx pods using two FADA volumes creating 40 volumes at same time on the node-1
+		    3. After that verify volumes are created successfully
+	*/
+	JustBeforeEach(func() {
+
+		StartTorpedoTest("FADAVolTokenTimout", "Validate FADA volumes token timeout when multiple requests hit same node at same time", nil, 0)
+	})
+	stepLog := "Deploy and attach multiple FADA volumes on the same node and validate token request crash"
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+		//Scheduling app with volume placement strategy
+		applist := Inst().AppList
+		rand.Seed(time.Now().Unix())
+		storageNodes := node.GetStorageNodes()
+		selectedNode := storageNodes[rand.Intn(len(storageNodes))]
+		var err error
+		defer func() {
+			Inst().AppList = applist
+			err = Inst().S.RemoveLabelOnNode(selectedNode, k8s.NodeType)
+			log.FailOnError(err, "error removing label on node [%s]", selectedNode.Name)
+		}()
+		Inst().AppList = []string{"nginx-fada-fastpath-vps"}
+		err = Inst().S.AddLabelOnNode(selectedNode, k8s.NodeType, k8s.FastpathNodeType)
+		log.FailOnError(err, fmt.Sprintf("Failed add label on node %s", selectedNode.Name))
+
+		stepLog = "Schecule apps and attach 200+ volumes"
+		i := 0
+		Step(stepLog, func() {
+			contexts = make([]*scheduler.Context, 0)
+			appScale := 110 //here it is 20 as each app deploys 2 volumes
+
+			i := 0
+			for i = 1; i < appScale; i++ {
+				contexts = append(contexts, ScheduleApplications(fmt.Sprintf("fadavoltkn-%d", i))...)
+			}
+			ValidateApplications(contexts)
+		})
+
+		var wg sync.WaitGroup
+
+		stepLog = "Attaching 40 volumes at same time"
+		scheduleCount := 20 //here it is 20 as each app deploys 2 volumes
+		Step(stepLog, func() {
+			scheduleAppParallel := func() {
+				defer wg.Done()
+				defer GinkgoRecover()
+				contexts = append(contexts, ScheduleApplications(fmt.Sprintf(fmt.Sprintf("fadavoltkn-%d", i)))...)
```

Comment:
What's the logic for using two fmt.Sprintf(fmt.Sprintf("fadavoltkn-%d", i))?

Reply:
it is just to create to have different namespaces